x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -176,6 +176,7 @@ export interface HostResultList {
   query_strategy_version: MetadataQueryStrategyVersions;
   /* policy IDs and versions */
   policy_info?: HostInfo['policy_info'];
+  host_status?: HostStatus;
```

Comment:
Curious: was this missing? or was it introduced recently and part of an merge upstream?

Reply:
i believe it was in the hostInfo type (aka the type for the endpoint list) but not for the details